mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -62,21 +70,29 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	FIELD f_naslivlb lastRenderY D
 	FIELD f_nexrnbri wasInPowderSnow Z
 	FIELD f_nzehfinv dimensions Lnet/minecraft/unmapped/C_sszpscpo;
+	FIELD f_oanvevib TOTAL_AIR_SUPPLY I
 	FIELD f_ocqvsaqw collisionBlockState Lnet/minecraft/unmapped/C_txtbiemp;
+	FIELD f_oritxgsn crystalSoundIntensity F
 	FIELD f_oysssbsf ridingCooldown I
 	FIELD f_paqudcks UUID_KEY Ljava/lang/String;
 	FIELD f_pebbtyzx NO_GRAVITY Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_pfezymmj prevYaw F
+	FIELD f_pfgaedzm BASE_TICKS_TO_FREEZE I
 	FIELD f_qazqhmfv SNEAKING_FLAG_INDEX I
 	FIELD f_qriixkyh SWIMMING_FLAG_INDEX I
 	FIELD f_qxfvmjxc lastRenderX D
 	FIELD f_qxqqtzyy NAME_VISIBLE Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_rbybyzcg flyDist F
 	FIELD f_rilctjct dataTracker Lnet/minecraft/unmapped/C_pyoaoolj;
+	FIELD f_rkfixhau verticalCollisionBelow Z
+	FIELD f_rnuwpthh WATER_FLOW_SCALE D
 	FIELD f_scozqngs invulnerable Z
 	FIELD f_sfjnvyan movementMultiplier Lnet/minecraft/unmapped/C_vgpupfxx;
 	FIELD f_skorbhua lastChimeAge I
 	FIELD f_smuuqtsj FROZEN_TICKS Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_srfebjdd DEFAULT_SHAPE_HEIGHT F
 	FIELD f_ssljujsr LOGGER Lorg/slf4j/Logger;
+	FIELD f_tigrjjuw DEFAULT_SHAPE_WIDTH F
```

Comment:
I think `BB` in mojmap, in this context means bounding box, not shape
```suggestion
	FIELD f_srfebjdd DEFAULT_BOUNDING_BOX_HEIGHT F
	FIELD f_ssljujsr LOGGER Lorg/slf4j/Logger;
	FIELD f_tigrjjuw DEFAULT_BOUNDING_BOX_WIDTH F
```

Reply:
ah right whoops